retry/retry.go

```diff
@@ -136,6 +136,11 @@ func ResetTimeout(t *time.Timer, d time.Duration) {
 // i.e. temporary, timeout, DNS, connection refused and reset, host down and unreachable and
 // network down and unreachable errors. In addition, any database error is considered retryable.
 func Retryable(err error) bool {
+	if errors.Is(err, context.DeadlineExceeded) {
+		// context.DeadlineExceeded is considered temporary and timeout, but we don't want to retry on it.
```

Comment:
This sentence looks wrong. Is it temporary and what about the timeout part? Could you please rephrase it.